episodes/04-changes.md

```diff
@@ -20,6 +20,12 @@ exercises: 0
 
 ::::::::::::::::::::::::::::::::::::::::::::::::::
 
+:::::::::::::::::::::::::::::::::::::::: instructor
+We suggest to use slides. From slide 5 onwards:
```

Comment:
I don't agree with this suggestion to be honest. I think it is confusing. I know it works for Ole, but I think it is better to leave it to the instructor how exactly they want to use the slides. What do you think?

Reply:
I agree that instructors should decide for themselves which slides to show and when, I personally found it difficult to navigate in between slides and command line at the beginning, so therefore this suggestion. I can remove it as well :)